bodhi/server/models.py

```diff
@@ -2571,7 +2576,7 @@ def requested_tag(self):
             # release to the Release.dist-tag
             if self.release.state is ReleaseState.pending:
                 tag = self.release.dist_tag
-        elif self.request is UpdateRequest.testing:
+        elif self.request in (UpdateRequest.testing, UpdateRequest.batched):
```

Comment:
(I made this comment on the fedora devel list and was told to post it here).

It would be nice if new package updates were also sent directly to stable with something like ```or self.type is newpackage```-- there isn't a compelling reason to batch new package updates (since they won't automatically get installed on an end-user system) and this way they don't get artificially made them any slower than they already are, ensuring brand-new packages still get to users in a timely manner.